### DISCUSSION TITLE
Edge Configuration

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#106] [Q&A]
Hello! I figured I would make a new discussion to compartmentalize. What does "Due to security reasons, many policies cannot be used when you are signed into Edge browser using personal Microsoft account. This script does not use any of those policies. When those policies are applied, they are ignored by the browser and edge://policy/ shows an error for them." mean? Should I not login into edge with my MPA?

**Comment by HotCakeX** [comment]
Hi, good question,
In this webpage where every policy for Edge browser is listed,
https://learn.microsoft.com/en-us/deployedge/microsoft-edge-policies

There is this notice

![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/b3caa319-9b6a-40eb-92a0-65a86c8ab39f)

If a policy has this

![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/41b6c812-63d6-4e8b-8e9a-b6592f78b21f)

Then it means it can be used when signing in Edge using personal Microsoft account

But if the policy has this

![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/dfb48d04-78ca-4a2f-9d8c-c6791787b6ef)

Then it can only be used when you sign into Edge with Microsoft Entra ID. They are mostly security related policies, like turning off/on SmartScreen etc.

The script doesn't apply policies that can't be used with a personal Microsoft account (MSA)

**Reply by ghost** [reply]
Oh perfect thank you! Do you recommend signing into edge with a personal account?

**Reply by HotCakeX** [reply]
Yes of course, sync your Edge settings so you don't have to apply them every time on a new browser. sync favorites/bookmarks, passwords etc. to keep them safe
